manifests/istio/istio/base/kustomization.yaml

```diff
@@ -16,13 +16,18 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - github.com/kubeflow/manifests/istio/istio/base?ref=v1.2-branch
+- github.com/kubeflow/manifests/istio/istio/overlays/https-gateway?ref=v1.2-branch
```

Comment:
Instead of referring to the istio base, here we just refer to the https-gateway overlay